packages/product-json-to-csv/test/map-headers.spec.js

```diff
@@ -1,181 +1,275 @@
 import mapHeaders from '../src/map-headers'
 
 describe('mapHeaders', () => {
-  it('maps headers from user', () => {
-    const headers = [
-      '_published',
-      '_hasStagedChanges',
-      'productType',
-      'variantId',
-      'variantKey',
-      'id',
-      'key',
-      'sku',
-      'tax',
-      'categories',
-      'images',
-      'name.en',
-      'description.en',
-      'slug.en',
-      'attribute',
-      'customAttribute',
-    ]
-    const expected = [
-      {
+  describe('maps headers from user', () => {
+    let headerFields
+    beforeEach(() => {
+      headerFields = [
+        '_published',
+        '_hasStagedChanges',
+        'productType',
+        'variantId',
+        'variantKey',
+        'id',
+        'key',
+        'sku',
+        'tax',
+        'categories',
+        'images',
+        'name.en',
+        'description.en',
+        'slug.en',
+        'attribute',
+        'customAttribute',
+      ]
+    })
+
+    it('contains `_published`', () => {
+      expect(mapHeaders(headerFields)[0]).toEqual({
         label: '_published',
         value: 'publish',
-      },
-      {
+      })
+    })
+
+    it('contains `_hasStagedChanges`', () => {
+      expect(mapHeaders(headerFields)[1]).toEqual({
         label: '_hasStagedChanges',
         value: 'hasStagedChanges',
-      },
-      {
+      })
+    })
+
+    it('contains `productType`', () => {
+      expect(mapHeaders(headerFields)[2]).toEqual({
         label: 'productType',
         value: 'productType',
-      },
-      {
+      })
+    })
+
+    it('contains `variantId`', () => {
+      expect(mapHeaders(headerFields)[3]).toEqual({
         label: 'variantId',
         value: 'variant.id',
-      },
-      {
+      })
+    })
+
+    it('contains `variantKey`', () => {
+      expect(mapHeaders(headerFields)[4]).toEqual({
         label: 'variantKey',
         value: 'variant.key',
-      },
-      {
+      })
+    })
+
+    it('contains `id`', () => {
+      expect(mapHeaders(headerFields)[5]).toEqual({
```

Comment:
In case you want this to me more resiliant why not go for a 

```js
expect(mapHeaders(headerFields)).toEqual(expect.arrayContaining([{ label: 'id', value: 'id' }])
```

or does the position matter so much? I guess not...

Reply:
The position matters. We had issues before that users complained the order of the fields in the templates didn't match the order in the exported file and this isn't the desired functionality